styles/learn-x-in-y-minutes/catppuccin.user.css

```
@@ -0,0 +1,208 @@
+    @lavender: @catppuccin[ @@lookup][ @lavender];
+    @text: @catppuccin[ @@lookup][ @text];
+    @subtext1: @catppuccin[ @@lookup][ @subtext1];
+    @subtext0: @catppuccin[ @@lookup][ @subtext0];
+    @overlay2: @catppuccin[ @@lookup][ @overlay2];
+    @overlay1: @catppuccin[ @@lookup][ @overlay1];
+    @overlay0: @catppuccin[ @@lookup][ @overlay0];
+    @surface2: @catppuccin[ @@lookup][ @surface2];
+    @surface1: @catppuccin[ @@lookup][ @surface1];
+    @surface0: @catppuccin[ @@lookup][ @surface0];
+    @base: @catppuccin[ @@lookup][ @base];
+    @mantle: @catppuccin[ @@lookup][ @mantle];
+    @crust: @catppuccin[ @@lookup][ @crust];
+    @accent-color: @catppuccin[ @@lookup][ @@accent];
+
+    --ctp-rosewater: @rosewater;
+    --ctp-flamingo: @flamingo;
+    --ctp-pink: @pink;
+    --ctp-mauve: @mauve;
+    --ctp-red: @red;
+    --ctp-maroon: @maroon;
+    --ctp-peach: @peach;
+    --ctp-yellow: @yellow;
+    --ctp-green: @green;
+    --ctp-teal: @teal;
+    --ctp-sky: @sky;
+    --ctp-sapphire: @sapphire;
+    --ctp-blue: @blue;
+    --ctp-lavender: @lavender;
+    --ctp-text: @text;
+    --ctp-subtext1: @subtext1;
+    --ctp-subtext0: @subtext0;
+    --ctp-overlay2: @overlay2;
+    --ctp-overlay1: @overlay1;
+    --ctp-overlay0: @overlay0;
+    --ctp-surface2: @surface2;
+    --ctp-surface1: @surface1;
+    --ctp-surface0: @surface0;
+    --ctp-base: @base;
+    --ctp-mantle: @mantle;
+    --ctp-crust: @crust;
+
+    color-scheme: if(@lookup =latte, light, dark);
+
+    ::selection {
+      background-color: fade(@accent-color, 30%);
+    }
+
+    &:not(.dark),
+    &:not(.light) {
+        p {
+          color: @subtext0 !important;
+        }
+            
+        span {
+          color: @text;
+        }
+    }
+
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    .share {
+      background-color: @mantle;
+      border-color: @crust;
+
+      &:hover {
+        background-color: @surface0;
+      }
+    }
+
+    .theme-choice button {
+      color: @text;
+      background-color: @mantle;
+      border-color: @crust;
+
+      &:hover {
+        background-color: @surface0;
+      }
+    }
+
+    #lang-select {
+      color: @text;
+      background-color: @mantle;
+      border-color: @crust;
+    }
+
+    td.lang span a {
+      color: @text;
+      background-color: @mantle;
+      border-color: @crust;
+
+      &:hover {
+        background-color: @surface0;
+      }
+    }
+
+    body, & {
+      background-color: @base;
+      color: @text;
+      border-color: @base;
+      box-shadow: none;
+    }
+
+    td,
+    th {
+      color: @text;
+    }
+
+    footer {
+      p {
+        color: @subtext1;
+      }
+    }
+
+    a {
+      color: @accent-color !important;
+      &:visited {
+        color: darken(@accent-color, 15%) !important;
+      }
+      &.active {
+        color: @base !important;
+        background-color: @accent-color !important!;
```

```suggestion
        background-color: @accent-color !important;
```
This might be causing the issue.